docs/source/api/adc.rst

```diff
@@ -16,8 +16,8 @@ ADCs are very useful in control and monitoring applications since most sensors
 Arduino-ESP32 ADC API
 ---------------------
 
-ADC common API
-**************
+ADC OneShot mode
+****************
 
```

Comment:
I think we should explain the basic differences between both modes.

Reply:
@pedrominatel Updated. Is this explanation understandable and enough? Thanks